bugsinpy_run_test.sh

```
@@ -0,0 +1 @@
+pytest tests/test_sessions.py::TestSession::test_download_in_session
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)